tests/trx_generator/trx_provider.cpp

```diff
@@ -88,11 +89,13 @@ namespace eosio::testing {
                                                     content_type};
       http_client_async::async_http_request(
           params, std::move(msg_body),
-          [&acked = _acknowledged](boost::beast::error_code                                      ec,
-                                   boost::beast::http::response<boost::beast::http::string_body> response) {
+          [this, &acked = _acknowledged, trx_id = trx.id()](
```

Comment:
Since you are capturing `this`, you don't need to explicitly capture `&acked`.

Reply:
Thanks. Had been planning to do this, but completely forgot to come back to it.
Addressed: [400c5b8](https://github.com/AntelopeIO/leap/pull/1165/commits/400c5b891a9f12e86916b246d1e4a073e39ec9e0)